tests/schema_update_test.go

```diff
@@ -438,7 +438,7 @@ func TestVCS(t *testing.T) {
 			err = ctl.vcsProvider.AddFiles(test.externalID, map[string]string{gitFile3: migrationStatement})
 			a.NoError(err)
 
-			payload, err := json.Marshal(test.newWebhookPushEvent([][]string{{gitFile}, {gitFile2}, {gitFile3}}, [][]string{nil, nil, nil}))
+			payload, err := json.Marshal(test.newWebhookPushEvent([][]string{{gitFile3}, {gitFile2}, {gitFile}}, [][]string{nil, nil, nil}))
```

Comment:
Maybe we should change the file name instead?

Seems strange that gitFile maps to book3.sql

Reply:
+1, done.